services/order.py

```diff
@@ -0,0 +1,29 @@
+from django.db import transaction
+from db.models import Order, Ticket
+
+def create_order(tickets, username, date=None):
+    with transaction.atomic():
+        order = Order.objects.create(user=username, created_at=date)
+
+        for ticket_data in tickets:
+            row = ticket_data.get("row")
+            seat = ticket_data.get("seat")
+            movie_session = ticket_data.get("movie_session")
+            ticket = Ticket.objects.create(
+                order=order,
+                row=row,
+                seat=seat,
+                movie_session=movie_session

```

Comment:
movie_session is id, not object
```suggestion
                movie_session_id=movie_session

```